web-ui/src/components/guides/GuideLink.jsx

```diff
@@ -0,0 +1,18 @@
+import PdfIcon from "@material-ui/icons/PictureAsPdf";
+import React from "react";
+
+const GuideLink = (props) => {
+    const path = "../../../pdfs/";
+    const fileName = props.name;
+    let fullPath = path + fileName + ".pdf";
+    return (
+        <li>
+            <PdfIcon />
+            <a href={fullPath} target="_blank">
```

Comment:
Pipeline check fails, it treats warning as errors, this is what it said.  

`Line 11:32:  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank`

Reply:
This passes when I run it locally. Do you know what check is being done on the server side so I don't have to wait until I push to catch these?

Reply:
Any warnings will be treated as errors in the pipeline, they are ignored in local runs. You can see warnings in the log on the console and you can set CI=true as an environment variable as well, this will cause warnings to fail the build.